src/containers/HomePage.js

```diff
@@ -60,40 +58,25 @@ class HomePage extends PureComponent {
     return this.props.addTarget(targetCompleted);
   }
 
-  MenuLeft() {
-    return this.state.isCreatingNewTarget ?
-      (
-        <div>
-          <div className="headerContent">
-            <FormattedMessage id="target.title.createTarget" />
-          </div>
-          <div className="content create-target">
-            <CreateTargetForm
-              onSubmit={this.handleCreateTarget}
-            />
-          </div>
-        </div>
-      ) :
-      (
-        <div>
-          <Welcome currentPage="Home" />
-          <div className="content">
-            <LogoutButton className="sign-in-button" />
-          </div>
-        </div>
-      );
-  }
-
   render() {
-    const { targetList, topicList } = this.props;
+    const { targetList, topicList, history } = this.props;
+    const { isCreatingNewTarget } = this.state;
+    const { location } = history;
+    let section = isCreatingNewTarget ? 'newTarget' : 'welcome';
+    section = location.pathname === '/about' ? 'aboutTarget' : section;
```

Comment:
you probably already have the `/about` route defined in routesPath right? perhaps use it from there?

Reply:
I didn't want to use this but perhaps it looks better!!

Reply:
const showMenu = location.pathname === '/about' && !isCreatingNewTarget;
    let section = isCreatingNewTarget ? 'newTarget' : 'welcome';
    section = location.pathname === '/about' ? 'aboutTarget' : section;

Reply:
hmm sorry for commenting the same again bellow. I had not seen this. Why didn't you want to do that?